src/components/TopNavDropdown/TopNavDropdownItem.vue

```diff
@@ -0,0 +1,47 @@
+
+<template>
+  <LobLink
+    :to="to"
+    :target="target"
+    :disabled="disabled"
+    :class="['flex flex-nowrap items-center no-underline py-2 px-4 type-small-500 hover:bg-gray-50',
+             { 'focus-visible:rounded-sm focus-visible:bg-gray-50 focus:outline-black focus:outline-dashed focus:!ring-none': !disabled },
+             { '!text-gray-500 disabled:!text-gray-300 hover:!text-gray-800 focus-visible:!text-gray-800 active:bg-black active:!text-white': !disabled },
```

Comment:
I was wondering about `disabled:!text-gray-300` being applied when `!disabled`. Is that correct?

Reply:
no that looks like a mistake, will take a look

Reply:
removed 👍🏼